src/cascadia/TerminalSettingsEditor/Profiles.cpp

```diff
@@ -3,13 +3,62 @@
 
 #include "pch.h"
 #include "Profiles.h"
+
```

Comment:
?

Reply:
Ah this is just me following the most common C++ style guide for imports... It breaks them up into the following order:
* "pch.h"
* "class.h"
* _blank line_
* \<system headers\>
* _blank line_
* \<stdlib headers\>
* _blank line_
* \<other global headers\>
* _blank line_
* "local headers.h"

(Google has a good explanation [here](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes).)

This ordering helps with clarity as you know immediately where some file resides at. This is especially helpful with MSVC, which for whatever reason has decided to not actually implement C++ and treats `#include "foo"` and `#include <foo>` synonymous. 😐 With the above order such imports cannot be confused anymore under msvc, which is nice.
You'll find that most projects follow this style and in fact even in this project most code follows this style already.

But in short: Considering that our project already does this for most files, I feel like it's alright to modify files that don't do this yet.